from exceptions.exception_validation_error import ValidationError

class InvalidPassword(ValidationError):
    pass